MaybeGame/EnemyStats.cs

```diff
@@ -0,0 +1,209 @@
+﻿using System;
+
+namespace MaybeGame
+{
+    public class EnemyStats
```

Comment:
Stats should be applicable to all kinds of entities. Use same Stats class for Player, Enemy, etc..

Reply:
Created a StatSettings.cs class